packages/mobile/src/fiatExchanges/CashInSuccess.tsx

```diff
@@ -14,19 +16,33 @@ import { StackParamList } from 'src/navigator/types'
 type RouteProps = StackScreenProps<StackParamList, Screens.CashInSuccess>
 type Props = RouteProps
 
+const capitalizeProvider = (provider?: string) => {
+  if (provider) {
+    const providerArr = provider.split('')
```

Comment:
not sure what this does. Did you mean split by the space character instead?

Reply:
This just splits every character into it's one item in an array

Reply:
A bit late to the conversation here, but would something like this have worked?
```JavaScript
const capitalizeProvider = (provider?: string) => {
  return provider && provider[0].toUpperCase() + provider.slice(1)
}
```